core/controllers/acl_decorators.py

```diff
@@ -1822,6 +1823,81 @@ def test_can_edit_skill(self, skill_id, **kwargs):
     return test_can_edit_skill
 
 
+def can_edit_skills(handler):
```

Comment:
I think we had a discussion on the actual user need for editing multiple skills at a time.

If this is not needed, shall we get rid of this decorator? I think it would be better to scope this sort of thing more narrowly.

Reply:
Done.